cmd/transformcalc.cpp

```diff
@@ -202,11 +215,23 @@ void run ()
     case 3: { // header
       if (num_inputs != 2)
         throw Exception ("header requires 2 inputs");
-      auto orig_header = Header::open (argument[0]);
-      auto modified_header = Header::open (argument[1]);
-
-      transform_type forward_transform = Transform(modified_header).voxel2scanner * Transform(orig_header).voxel2scanner.inverse();
-      save_transform (forward_transform.inverse(), output_path);
```

Comment:
Not sure why this composed the transform from modified header to original header and then inverted it, as opposed to just computing the transform from original heder to modified header...